test/functional/api/extmark_spec.lua

```diff
@@ -1712,6 +1712,10 @@ describe('API/extmarks', function()
           aaa bbb ccc                         |*2
                                               |
     ]])
+    -- decor is not removed twice
```

Comment:
For what it's worth, this test will likely be ineffective after #27011 (as it removes the assert).

Reply:
to make things simple, #27011 doesn't care about invalidation at all. but once the basic version is done and merged, we could consider it as a follow-up.

Reply:
Sure. To be clear, I was just commenting on the effectiveness of the added test here. Not that the fact that `mt_meta_total()` doesn't take into account invalid marks will necessarily result in problems. The fact that it doesn't just means that this test would hit a different assert without the fix in this PR.